boards/arm/efr32_slwstk6061a/efr32_slwstk6061a.dts

```diff
@@ -42,12 +42,12 @@
 		compatible = "gpio-keys";
 		button0: button_0 {
 			/* gpio flags need validation */
-			gpios = <&gpiof 6 GPIO_INT_ACTIVE_LOW>;
+			gpios = <&gpiof 6 GPIO_INT_LOW>;
```

Comment:
Wasn't the `GPIO_ACTIVE_LOW` flag supposed to be used also to specify active state of inputs. One may not want to use interrupts to handle buttons. Sometimes just reading the current state of the button could be desired.

Reply:
That's correct. I will change this line to
```
	gpios = <&gpiof 6 (GPIO_ACTIVE_LOW | GPIO_INT_LOW)>;
```